database/factories/AnimeFactory.php

```diff
@@ -55,7 +57,7 @@ public function definition(): array
             "favorites" => $this->faker->randomDigitNotNull(),
             "synopsis" => "test",
             "background" => "test",
-            "season" => $this->faker->randomElement(["winter", "spring", "fall", "summer"]),
+            "premiered" => $this->faker->randomElement(["Winter", "Spring", "Fall", "Summer"]),
```

Comment:
`season` is returned - not premiered.
Also `year` is missing here

Reply:
But we have `premiered` in the database, I've changed this after the test was complaining about `premiered doesn't exist in collection`. Will add `year`.

Reply:
@pushrbx Yes, `premiered` is from the parser which is the parsed string from MAL.
The REST API breaks it down into `season` and `year` here:
https://github.com/jikan-me/jikan-rest/blob/ad066c1c22ccb7740bd4176b91597ae658ba1aae/app/Anime.php#L51-L64

For schema reference use the [REST API docs](https://docs.api.jikan.moe/), not the DB.